packages/contentful--create-contentful-app/lib/utils.js

```diff
@@ -15,25 +15,19 @@ export function exec(command, args, options) {
   });
 }
 
-export function printHelpText(mainCommand, localCommand) {
-  console.log(`
-${chalk.bold(localCommand)}
-
-${chalk.dim('Available commands:')}
-
-${chalk.cyan(`$ ${mainCommand} init app-name`)}
-
-Bootstraps your app inside a new folder "app-name".
-
-${chalk.cyan(`$ ${mainCommand} create-definition`)}
-
-Creates an app definition for your app in a Contentful
-organization of your choice.
-`);
-}
-
 export function rmIfExists(path) {
   if (existsSync(path)) {
     rmSync(path);
   }
 }
+
+export function detectManager() {
+  switch (basename(process.env.npm_execpath || '')) {
```

Comment:
Is this something that is always available or do we need to set it?

Reply:
Probably it is always available, but if not we fallback to npm.

Reply:
`node -pe "process.env.npm_execpath"` is `undefined` for me. Can we confirm that this is something that's commonly available?

Reply:
it is available only if you use npm or yarn.

Reply:
Ah - good point. I see it also if I run it thru a script